src/ixdat/techniques/ec_ms.py

```diff
@@ -188,17 +192,28 @@ def ecms_calibration_curve(
             n_list.append(n)
         n_vec = np.array(n_list)
         Y_vec = np.array(Y_list)
-        pfit = np.polyfit(n_vec, Y_vec, deg=1)
-        F = pfit[0]
+        n_fit = np.array([0, max(n_vec)])
+        if force_through_zero:
```

Comment:
I have to say, I am quite surprised that np.polyfit() doesnt have this as an option (but I did google for it so I accept it).